src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -178,6 +178,8 @@ public class LivingEntityGUI extends ModElementGUI<LivingEntity> implements IBlo
 	private final JTextField mobLabel = new JTextField();
 
 	private final JCheckBox spawnInDungeons = L10N.checkbox("elementgui.living_entity.spawn_dungeons");
+
+	private final JCheckBox spawnInRaids = L10N.checkbox("elementgui.living_entity.spawn_raids");

```

Comment:
I'd move the space from above `spawnInRaids` to below it, so the checkboxes can be together, and separated from the JColors

Reply:
Done